lib/godmin/helpers/forms.rb

```diff
@@ -4,6 +4,11 @@ module Forms
       def form_for(record, options = {}, &block)
         super(record, { builder: FormBuilders::FormBuilder, inline_errors: false }.merge(options), &block)
       end
+
+      def select_tag(name, option_tags = nil, options = {})
```

Comment:
Perhaps we should have gone with a `godmin_form_for` and a `godmin_form_tag` instead of overriding `form_for`, but yeah..